core/indigo-core/reaction/pathway_reaction.h

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+ * Copyright (C) from 2009 to Present EPAM Systems.
+ *
+ * This file is part of Indigo toolkit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+#pragma once
+
+#ifdef _WIN32
+#pragma warning(push)
```

Comment:
1) It is not win32 specific - it is MSVC specific
2) where is "#pragma warning(pop)" ?

Reply:
1. You want it to be _MSC_VER? Perhaps, you're correct, but we use _WIN32 all over the code for this matter. And I won't change it in other places. Do you want it to be MSC_VER here and WIN32 in other places?
2. Fixed